content/sensu-go/6.7/observability-pipeline/observe-schedule/agent.md

```diff
@@ -1147,6 +1148,19 @@ sensu-agent start --log-level debug{{< /code >}}
 /etc/sensu/agent.yml example | {{< code shell >}}
 log-level: debug{{< /code >}}
 
+<a id="max-session-length-attribute"></a>
+
+| max-session-length |      |
+--------------|------
+description   | Maximum duration for any one agent connection. In milliseconds (`ms`), seconds (`s`), minutes (`m`), or hours (`h`). When a connection reaches the specified amount of time, Sensu will force the agent to disconnect and reconnect to an available configured backend to prevent agent connection distribution from becoming skewed over time. The max-session-length algorithm includes random jitter so that agents will not disconnect and reconnect all at once.
```

Comment:
```suggestion
description   | Maximum duration for any one agent connection. In milliseconds (`ms`), seconds (`s`), minutes (`m`), or hours (`h`). Use max-session-length to prevent agent connection distribution from becoming skewed over time.<br><br>The max-session-length algorithm includes random jitter so that agents will not disconnect and reconnect all at once. Based on the random jitter calculation, at some time before a connection reaches the specified maximum duration, Sensu will force the agent to disconnect and reconnect to an available configured backend.
```